flutter/shell/platform/tizen/system_utils.h

```diff
@@ -10,6 +10,14 @@
 
 namespace flutter {
 
+#if defined(MOBILE_PROFILE)
+constexpr double kProfileFactor = 0.7;
+#elif defined(TV_PROFILE)
+constexpr double kProfileFactor = 2.0;
+#else
+constexpr double kProfileFactor = 1.0;
+#endif
```

Comment:
These constants are only used in system_utils.cc, so they don't need to be declared in header. Please move them to system_utils.cc and wrap them in an unnamed namespace.

Reply:
OK. I have updated it.